src/electron.ts

```diff
@@ -30,27 +25,12 @@ function createWindow(): Electron.BrowserWindow {
   });
 
   win.setMenu(null);
-  win.loadURL(`file://${__dirname}/index.html`);
+  win.loadURL(`file://${__dirname}/index.html?version=${app.getVersion()}`);
```

Comment:
Yes, otherwise I would not know the version of app inside component. Only if there is other way to pass value from electron.ts to app.component.ts?